### PR TITLE
Install LFS filters

### LINKED ISSUE #2227: Configure Git LFS filter
Run `git lfs install` to set up the global LFS filter.

### PR DESCRIPTION
**Pull request by joshaber**
Fixes #2227 

1. Run `git lfs install --skip-repo` silently on launch to setup the global filters.
1. Ask the user if they want to run `git lfs install` to setup the hooks when adding a repository which uses LFS.

screen shot 2017-09-05 at 2 54 32 pm

### CODE REVIEW

**Comment by joshaber** [comment]
🎠

File: app/src/ui/lfs/initialize-lfs.tsx [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    if (this.props.repositories.length > MaxRepositoriesToList) {
+      return (
+        <p>
+          {this.props.repositories.length} repositories use{' '}
+          <LinkButton uri={LFSURL}>Git LFS</LinkButton>. To contribute to them,
+          Git LFS must first be initialized. Would you like to do so now?
+        </p>
+      )
+    } else {
+      const plural = this.props.repositories.length === 1
```
Comment (shiftkey) [inline app/src/ui/lfs/initialize-lfs.tsx]: Nit: the variable name `plural` uses `true` to indicate singular and `false` to indicate multiple and it's tearing me apart. What about a better name?

Comment (joshaber) [reply app/src/ui/lfs/initialize-lfs.tsx]: Looooooooooool

**Comment by joshaber** [comment]
🏭

**Comment by shiftkey** [comment]
#jolo

**Pull request merged**